### PR TITLE
Customer test

### PR DESCRIPTION
**Pull request by esdrasedu**
issue #2

### CODE REVIEW

**Comment by lucca65** [comment]
+1

File: test/kiik/customer_test.rb [hunk 1]
```diff
@@ -2,5 +2,45 @@
 
 module Kiik
   class CustomerTest < Test::Unit::TestCase
+
+    should "create empty customer should raise exception" do
```
Comment (leorog) [inline test/kiik/customer_test.rb]: hint: could improve the tests name to be more semantic. 
"should" + "create empty customer should raise exception" 
to
"should" + "raise exception for empty customer"

**Comment by leorog** [comment]
+ 1

File: test/kiik/customer_test.rb (hunk 1 above)
Comment (lucca65) [inline test/kiik/customer_test.rb]: Nice comment! the suggestion makes it more readable

**Pull request merged**